packages/web/src/components/nav/desktop/NowPlayingArtworkTile.tsx

```diff
@@ -157,6 +154,29 @@ export const NowPlayingArtworkTile = () => {
           </DynamicImage>
         </Link>
       </FadeInUp>
+    )
+  }
+
+  if (isPremium) {
```

Comment:
is there not a disabled prop we can pass into draggable or something? otherwise this looks god

Reply:
genius. let me check lol

Reply:
not doing what im expecting